### ISSUE TITLE
For windows users (additional info I found super helpful)

### ISSUE THREAD

**Issue by di212** [cyclus/cyclus#1564]
I had troubles setting up anaconda on Ubuntu on Windows and this GitHub page was very helpful https://gist.github.com/kauffmanes/5e74916617f9993bc3479f401dfec7da and when I had troubles with the python itself https://askubuntu.com/questions/1144446/python-installed-in-ubuntu-but-python-command-not-found

**Comment by gonuke** [comment]
Thanks for these tips, @di212.  Did this come up because you are trying to install Cyclus in Windows?  Are you using the Windows Subsystem for Linux (WSL)?

**Comment by di212** [comment]
Yes, I installed Ubuntu for to run cyclus on Windows and had a hard time with things not working as they should

**Comment by di212** [comment]
does not help
